portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Omar Montiel's Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>I am an aspiring Software Engineer, studying Computer Science and currently a Google Intern :)</p>
+      <p>I love video games, piano, and learning about God!</p>
+      <p>Click here to get one of my favorite Bible verses:</p>
+      <button onclick="addRandomBibleVerse()">Hello</button>
+      <div id="bible-verse-text-container"></div>
+      <div id="bible-verse-container"></div>
+      <p>I also snowboard:</p>
+      <img src="/images/portfolio-image.jpg"/>
```

Comment:
Make sure you add descriptive alt text! https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Reply:
Done!